Makefile

```diff
@@ -22,10 +22,13 @@ test-for-ga:
 	make test
 
 publish:
-	scripts/publish
+	# New Python based publish script in dcicutils (2023-04-25).
+	python -m dcicutils.scripts.publish_to_pypi
 
 publish-for-ga:
-	scripts/publish --noconfirm
+	# New Python based publish script in dcicutils (2023-04-25).
+	pip install dcicutils==7.3.0.1b17
```

Comment:
If this library is installed, its script should get set up if you've declared the script right in pyproject.toml. See remarks above.

Reply:
This is all set now.